src/line-number-gutter-component.coffee

```diff
@@ -1,13 +1,13 @@
-_ = require 'underscore-plus'
-{setDimensionsAndBackground} = require './gutter-component-helpers'
-
+TiledComponent = require './tiled-component'
+LineNumbersTileComponent = require './line-numbers-tile-component'
 WrapperDiv = document.createElement('div')
+DummyLineNumberComponent = LineNumbersTileComponent.createDummy()
 
 module.exports =
-class LineNumberGutterComponent
+class LineNumberGutterComponent extends TiledComponent
   dummyLineNumberNode: null
 
-  constructor: ({@onMouseDown, @editor, @gutter}) ->
+  constructor: ({@onMouseDown, @editor, @presenter, @gutter}) ->
```

Comment:
We should avoid passing the `presenter` around. Are you planning to refactor this later?